components/Hero/src/index.scss

```diff
@@ -147,6 +147,95 @@
+  clip-path: var(--denhaag-hero-theme-image-content-clip-path);
+  padding-block-end: var(--denhaag-hero-theme-image-content-padding-block-end);
+  padding-block-start: var(--denhaag-hero-theme-image-content-padding-block-start);
+  padding-inline-end: var(--denhaag-hero-theme-image-content-padding-inline);
+  padding-inline-start: var(--denhaag-hero-theme-image-content-padding-inline);
+  text-align: var(--denhaag-hero-theme-image-content-text-align);
+}
+
+.denhaag-hero--theme-image .denhaag-hero__image {
+  bottom: var(--denhaag-hero-theme-image-image-bottom);
+  clip-path: var(--denhaag-hero-theme-image-image-clip-path);
+  width: var(--denhaag-hero-theme-image-image-width);
+  height: var(--denhaag-hero-theme-image-image-height);
+  left: var(--denhaag-hero-theme-image-image-left);
+  object-position: var(--denhaag-hero-theme-image-image-object-position);
+
+  @media (min-width: 768px) {
```

Comment:
Verplaatsen naar CSS variabelen die gezet worden in de `.denhaag-hero--theme-image`, dit wordt dan o.a. op de manier die al in deze file geüpdatet. (zie https://github.com/nl-design-system/denhaag/pull/1202/files#diff-de06012202559ddbcd803978026c66ff8e0d265ee113f259d4320b8e6648b400R250 )